### DISCUSSION TITLE
Theme-selection ignored on Windows 11?

### DISCUSSION THREAD

**Discussion by CruiseMaster** [Simnico99/MicaWPF#94] [General]
Hi,
I've set the application-theme to "dark" in App.xaml like:
`<mica:ThemeDictionary DesignModeTheme="Dark" Theme="Dark" />`
but if Windows is set to light-mode, my application uses light-mode as well even though it's set to "Dark".
I want to force the application to be in dark-mode whatever Windows is set to.
Am I missing something?
I'm using the newest Windows 11 with .NET 8.0.1.
Thanks in advance. :)

**Comment by Simnico99** [comment]
Opened an issue and a pull request
https://github.com/Simnico99/MicaWPF/issues/95
https://github.com/Simnico99/MicaWPF/pull/96

Released fixed version:
https://github.com/Simnico99/MicaWPF/releases/tag/6.2.6

Please wait for the pipeline to finish and the package to upload.

**Comment by CruiseMaster** [comment]
Wow, that was quick!
Thank you!
I've tested it and it works.
